validate.sh

```diff
@@ -110,6 +110,11 @@ validate_example "cloud/openfaas/thumbnail-generator/image-resize" "service.yaml
 validate_example "cloud/platform-connection/aws-azure-connection/aws-azure" "service.yaml" "inputs.yaml"
 validate_example "cloud/platform-connection/aws-azure-connection/azure-aws" "service.yaml" "inputs.yaml"
 
+# test an example from kubernetes/docker
+validate_example "kubernetes/docker" "service.yaml" "inputs.yaml"
+# test an example from kubernetes/rancher
+validate_example "kubernetes/docker" "service.yaml" "inputs.yaml"
```

Comment:
The last line very likely needs to be kubernetes/rancher, right?

Reply:
True, will fix.

Reply:
Fixed.